packages/fxa-payments-server/src/routes/Checkout/index.scss

```diff
@@ -37,6 +37,29 @@
   }
 }
 
+.legal-blurb {
+  clear: both;
+  color: #686869;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  text-align: center;
+
+  p {
+    margin-bottom: 8px;
+    margin-top: 0;
+  }
+
+  p:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.legal-heading {
+  font-weight: 600;
+  margin-top: 8px !important;
+}
+
```

Comment:
These styles should be at the component level once these links are moved inside a component (like `TermsAndPrivacy`).